cmd/jsonnet.cpp

```diff
@@ -110,8 +110,8 @@ void usage(std::ostream &o)
     o << "  -h / --help             This message\n";
```

Comment:
Can you change the help message to better promote the ability to use multiple filenames (I also tweaked some of the text here):

```
General commandline:
jsonnet [<cmd>] {<option>} { <filename> }
Note: <cmd> defaults to "eval".
```

```
The eval command:
jsonnet eval {<option>} <filename>
Note: Only one filename is supported
```

```
The fmt command:
jsonnet fmt {<option>} { <filename> }
Note: Some options do not support multiple filenames.
```

And then remove the line `Note: Multiple filenames can be provided at once when using -i or --test options.`